### ISSUE TITLE
model.save(url) isn't saving weights file

### ISSUE THREAD

**Issue by BradHacker** [tensorflow/tfjs#513]
Let me preface this by saying I am a beginner in the TensorFlow community, although I am familiar with the concepts. This is my first project with TensorFlow JS.

#### TensorFlow.js version
0.12.0
#### Browser version
Chrome Version 67.0.3396.99
#### Describe the problem or feature request
I am running this webpage with [http-server](https://www.npmjs.com/package/http-server).
I am using `model.save()` to save the model to local storage, but it only saves a JSON file and never saves the weights.bin file. Now when I try and load the model, it can't find the weights file becuase it was never created.

#### Code to reproduce the bug / link to feature request
```javascript
async function train() {
  const options = {
    epochs: epochs,
    validationSplit: 0.1,
    shuffle: true,
    callbacks: {
      onTrainBegin: () => {
        console.log('training start')
        lossVals = []
      },
      onTrainEnd: () => {
        console.log('training complete')
        saveModel().then((results) => {
          console.log(results)
        })
      },
      onEpochEnd: async (num, logs) => {
        lossVals.push({
          epoch: num,
          loss: logs.val_loss
        })
        drawLoss(lossVals)
        if (!lowest_loss || logs.val_loss < lowest_loss.loss) {
          lowest_loss = {
            epoch: num,
            loss: logs.val_loss
          }
        }
        console.log('Epoch: ' + num)
        console.log('Loss: ' + logs.val_loss)
        $('#loss').text('latest loss: ' + logs.val_loss + '   |  epoch ' + lowest_loss.epoch + ' had the lowest loss of ' + lowest_loss.loss)
        await tf.nextFrame()
      }
    }
  }

  return model.fit(xs, ys, options)
}

async function saveModel() {
  return await model.save(`downloads://${hidden_units}_hidden_units_${epochs}_epochs_${lr*100}_lr`)
}
```

**Comment by caisq** [comment]
Hi @BradHacker , as a diagnostic step, can you go to this link below and run "Example 3?" Let me know whether it downloads one or two files.

https://js.tensorflow.org/api/0.12.0/#tf.Model.save

**Comment by BradHacker** [comment]
I use this code: 
```html
<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>TensorFlow Test</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@0.12.0"> </script>
</head>
<body>
  <script>
    const model = tf.sequential(
      {layers: [tf.layers.dense({units: 1, inputShape: [3]})]});
    const saveResults = model.save('downloads://my-model-1');
  </script>
</body>
</html>
```
It still only downloads one file called `my-model-1`

**Comment by caisq** [comment]
@BradHacker You didn't answer my question about how many files are downloaded when you run Example 3 at https://js.tensorflow.org/api/0.12.0/#tf.Model.save

Some browser settings will prevent >1 file to be downloaded at the same time. Can you check that your browser is not set like that? For more details see:
https://ccm.net/faq/32938-google-chrome-allow-websites-to-perform-simultaneous-downloads

**Comment by BradHacker** [comment]
It only downloads **one file**. My chrome settings should allow multiple files to be downloaded. I attached a screenshot of my automatic download settings in chrome.
![image](https://user-images.githubusercontent.com/22984089/42719012-840e2e78-86dd-11e8-9473-2f4edbbdcc14.png)

**Comment by BradHacker** [comment]
So it seems this might be an issue with chrome because there is a number on the chrome icon signifying active downloads, but when I go to the downloads page (Mac: Command + Shift + J), there are no active downloads and the number doesn't go away when I clear my downloads list. So the download for the weights file seems to be initiated, but it never finishes.

Using macOS High Sierra 10.13.5

**Comment by davidsoergel** [comment]
That does sound like there may be something screwy with your Chrome.  Can you try reinstalling it?

**Comment by davidsoergel** [comment]
Oh, also: You mentioned "one file called my-model-1".  Did you mean my-model-1.json?

**Comment by BradHacker** [comment]
Yes I meant to say my-model-1.json. Reinstalling chrome didn't fix the issue.

**Comment by davidsoergel** [comment]
Hm, that is puzzling.  Is there any chance that you have some other process removing files from your downloads directory?  E.g. a virus checker might be spuriously quarantining my-model-1.weights.bin, or a tool like Hazel might be automatically sorting your downloads into folders, etc.

Does the downloads list indicate that the weights file was downloaded, or does it just not show up there at all?  E.g., after running Example 3 that @caisq pointed at, mine looks like this:

![screen shot 2018-07-25 at 9 42 16 am](https://user-images.githubusercontent.com/1347381/43204527-210c2f44-8fef-11e8-9cf4-3f4dbd3e9c23.png)

**Comment by BradHacker** [comment]
I tried it twice. The bottom download being with running on http-server and the second was just navigating to the `file:///` path of the html file. When navigating to the file I was asked if I wanted to allow multiple files to download and I selected 'Allow', yet it yielded no new results. My chrome icon now says I have 2 new unfinished downloads to make a total of 5 since I've started chrome.

![image](https://user-images.githubusercontent.com/22984089/43213266-35cee0b0-9004-11e8-8f26-9266aa7fb4f0.png)
![image](https://user-images.githubusercontent.com/22984089/43213536-d3cc6468-9004-11e8-8bb2-dab5b95729a0.png)

**Comment by playgrdstar** [comment]
I had the same problem, but managed to resolve it with the following steps:

- Go to chrome://settings >> Content Settings >> Automatic downloads; or just go straight to chrome://settings/content/automaticDownloads 
- Look under 'Allow' for the address of the local file where you are running the tensorflow.js scripts
- Remove the address
- Train your tensorflow model again
- There will be a prompt for you to download the json file
- Followed by a pop-up asking for permission to download multiple files
- Click 'Allow'
- There will now be  prompt for you to download the bin file with the weights.
